judgels-frontends/raphael/src/routes/contests/contests/single/clarifications/modules/contestClarificationActions.js

```diff
@@ -37,12 +38,15 @@ export function answerClarification(contestJid, clarificationJid, answer) {
   };
 }
 
-export function alertNewClarifications(status) {
+export function alertNewClarifications(status, notificationTag) {
   return async () => {
+    let message;
     if (status === ContestClarificationStatus.Answered) {
-      toastActions.showAlertToast('You have new answered clarification(s).');
+      message = 'You have new answered clarification(s).';
     } else {
-      toastActions.showAlertToast('You have new clarification(s).');
+      message = 'You have new clarification(s).';
     }
+    toastActions.showAlertToast(message);
+    showDesktopNotification('TLX Clarification', notificationTag, message);
```

Comment:
```suggestion
    showDesktopNotification('New clarification(s)', notificationTag, message);
```

Should we also distinguish between answered and new clarifications here?

Reply:
Hmmm yes, I think that's a good idea.

Reply:
I distinguish it by adding 'Answered' to the string:
- "New answered clarification(s)"
- "New clarification(s)"